### ISSUE TITLE
Trying to run it on Windows 10 with simplest config

### ISSUE THREAD

**Issue by Masood-Lapeh** [Doridian/wsvpn#115]
```
PS C:\Users\---\Desktop> .\wsvpn-windows-amd64.exe --mode=client --config=wsvpn.yaml
2022/10/31 21:33:58 Local version is: v5.18.1 (protocol 12)
2022/10/31 21:33:58 base.go:99: [CLIENT-UNSET] WARNING: Connecting to ws://----:9900 with authentications: NONE
2022/10/31 21:33:59 connection.go:55: [CLIENT-UNSET] Command serialization: json
2022/10/31 21:33:59 connection.go:61: [CLIENT-UNSET] Unencrypted WebSocket connection established
2022/10/31 21:33:59 commands.go:62: [CLIENT-UNSET] Remote version is: v5.18.1 (protocol 12)
2022/10/31 21:33:59 base.go:169: [CLIENT-UNSET] Setting fragmentation: enabled
2022/10/31 21:33:59 commands.go:82: [CLIENT-UNSET] Got reply to command ID 13cd9378-f35d-40ee-a42a-454d10123c8b (ok): OK
2022/10/31 21:33:59 commands.go:67: [CLIENT-baea8a43-55a3-4e62-88ca-fc15e98e600c] Network mode TUN, Subnet 192.168.3.2/24, MTU 1420, IPConfig enabled
2022/10/31 21:34:00 Using existing driver 0.14
2022/10/31 21:34:00 Creating adapter
2022/10/31 21:34:01 commands.go:85: [CLIENT-baea8a43-55a3-4e62-88ca-fc15e98e600c] Opened interface WaterWinTunInterface

 OK!
2022/10/31 21:34:04 commands.go:109: [CLIENT-baea8a43-55a3-4e62-88ca-fc15e98e600c] Configured interface, VPN online
2022/10/31 21:34:24 ping.go:37: [CLIENT-baea8a43-55a3-4e62-88ca-fc15e98e600c] Sent ping
2022/10/31 21:34:24 ping.go:13: [CLIENT-baea8a43-55a3-4e62-88ca-fc15e98e600c] Received pong
2022/10/31 21:34:49 ping.go:37: [CLIENT-baea8a43-55a3-4e62-88ca-fc15e98e600c] Sent ping
2022/10/31 21:34:49 ping.go:13: [CLIENT-baea8a43-55a3-4e62-88ca-fc15e98e600c] Received pong
```

Now it seems OK, but when in browser I check my IP address in some websites, it's not changed. Maybe it has something to do with my previous run:

```
PS C:\Users\---\Desktop> .\wsvpn-windows-amd64.exe --mode=client --config=wsvpn.yaml
2022/10/31 21:23:20 Local version is: v5.18.1 (protocol 12)
2022/10/31 21:23:20 base.go:99: [CLIENT-UNSET] WARNING: Connecting to ws://---:9900 with authentications: NONE
2022/10/31 21:23:20 connection.go:55: [CLIENT-UNSET] Command serialization: json
2022/10/31 21:23:20 connection.go:61: [CLIENT-UNSET] Unencrypted WebSocket connection established
2022/10/31 21:23:20 commands.go:62: [CLIENT-UNSET] Remote version is: v5.18.1 (protocol 12)
2022/10/31 21:23:20 base.go:169: [CLIENT-UNSET] Setting fragmentation: enabled
2022/10/31 21:23:21 commands.go:82: [CLIENT-UNSET] Got reply to command ID 8f10f237-a6ec-400b-bb4a-04d84ff5314e (ok): OK
2022/10/31 21:23:21 commands.go:67: [CLIENT-2d4dc066-1a38-45c5-a495-99d873e8ba3a] Network mode TUN, Subnet 192.168.3.2/24, MTU 1420, IPConfig enabled
2022/10/31 21:23:34 Waiting for existing driver to unload from kernel
2022/10/31 21:23:34 Removing existing driver 0.8
2022/10/31 21:23:42 Installing driver 0.14
2022/10/31 21:23:42 Extracting driver
2022/10/31 21:23:42 Installing driver
2022/10/31 21:23:44 Creating adapter
2022/10/31 21:23:45 ping.go:37: [CLIENT-2d4dc066-1a38-45c5-a495-99d873e8ba3a] Sent ping
2022/10/31 21:23:50 ping.go:45: [CLIENT-2d4dc066-1a38-45c5-a495-99d873e8ba3a] Ping timeout
2022/10/31 21:23:54 Removed orphaned adapter "wintun"
2022/10/31 21:23:54 commands.go:85: [CLIENT-2d4dc066-1a38-45c5-a495-99d873e8ba3a] Opened interface WaterWinTunInterface

The route addition failed: The object already exists.

2022/10/31 21:24:33 commands.go:109: [CLIENT-2d4dc066-1a38-45c5-a495-99d873e8ba3a] Configured interface, VPN online
2022/10/31 21:24:33 vpnnet.go:48: [CLIENT-2d4dc066-1a38-45c5-a495-99d873e8ba3a] Error reading packet from tun: file already closed
2022/10/31 21:24:35 base.go:67: [CLIENT-2d4dc066-1a38-45c5-a495-99d873e8ba3a] Automatic reconnection disabled, exiting!
```

Client's confguration:

```
client:
    server: "ws://---:9900"
```

Server's configuration:

```
tunnel:
    subnet: 192.168.3.0/24

server:
    listen: "0.0.0.0:9900"
```

**Comment by Doridian** [comment]
WSVPN by default does not take over your default gateway by design. If you want to do that, you need to add the below to your client configuration:

```yaml
tunnel:
  set-default-gateway: false
```

Let me know if that fixes the issue for you.

The `The route addition failed: The object already exists.` is odd but Windows can be weird with that sometimes (the route in question is the one for `192.168.3.0/24` that it adds for the subnet).

**Comment by Masood-Lapeh** [comment]
@Doridian Thanks for your fast reply, excuse me for my late reply! I restarted Windows and retried what you've suggested. But still the same issue. (I can't work with Maple proxy too, unlike before. Maybe there is something happened to my Windows.)

Client's config:

```
client:
    server: "ws://o-----------r:9900"
tunnel:
    set-default-gateway: true
```

Client's output:

```
C:\Users\---------\Desktop>.\wsvpn-windows-amd64.exe --mode=client --config=wsvpn.yaml
2022/11/09 12:57:50 Local version is: v5.18.1 (protocol 12)
2022/11/09 12:57:50 base.go:99: [CLIENT-UNSET] WARNING: Connecting to ws://o-----------r:9900 with authentications: NONE
2022/11/09 12:57:50 connection.go:55: [CLIENT-UNSET] Command serialization: json
2022/11/09 12:57:50 connection.go:61: [CLIENT-UNSET] Unencrypted WebSocket connection established
2022/11/09 12:57:50 commands.go:62: [CLIENT-UNSET] Remote version is: v5.18.1 (protocol 12)
2022/11/09 12:57:50 base.go:169: [CLIENT-UNSET] Setting fragmentation: enabled
2022/11/09 12:57:50 commands.go:82: [CLIENT-UNSET] Got reply to command ID 190ea5b9-a73b-47a3-8432-e72cf6be54e8 (ok): OK
2022/11/09 12:57:50 commands.go:67: [CLIENT-b79933be-9058-4069-a28d-2b9c04814f4e] Network mode TUN, Subnet 192.168.3.2/24, MTU 1420, IPConfig enabled
2022/11/09 12:57:52 Using existing driver 0.14
2022/11/09 12:57:52 Creating adapter
2022/11/09 12:57:53 commands.go:85: [CLIENT-b79933be-9058-4069-a28d-2b9c04814f4e] Opened interface WaterWinTunInterface

 OK!
2022/11/09 12:57:55 commands.go:109: [CLIENT-b79933be-9058-4069-a28d-2b9c04814f4e] Configured interface, VPN online
2022/11/09 12:58:15 ping.go:37: [CLIENT-b79933be-9058-4069-a28d-2b9c04814f4e] Sent ping
2022/11/09 12:58:15 ping.go:13: [CLIENT-b79933be-9058-4069-a28d-2b9c04814f4e] Received pong
2022/11/09 12:58:40 ping.go:37: [CLIENT-b79933be-9058-4069-a28d-2b9c04814f4e] Sent ping
2022/11/09 12:58:40 ping.go:13: [CLIENT-b79933be-9058-4069-a28d-2b9c04814f4e] Received pong
...
```

**Comment by Onair-santa** [comment]
Yes. Default-gateway not work in Win.

**Comment by Doridian** [comment]
Found an issue with the default gateway, fixed in v5.31.0.

If this issue is still happening, please re-open it so I can see it easier. (GitHub does not show me closed issues easily)
